service-node/shared/src/main/scala/org/enterprisedlt/fabric/service/node/shared/Model.scala

```diff
@@ -155,6 +155,17 @@ object ChainCodeInfo {
     implicit val rw: RW[ChainCodeInfo] = macroRW
 }
 
+// ------------------------------------------------------------------------
+case class ApplicationInfo(
```

Comment:
seems like we don't need this separate entity

Reply:
we do need it, as we will se what applications are created.
Same as ChainCodeInfo object but for Applications.